benefit-finder/cypress/e2e/death-of-a-loved-one/selected-criteria-eligibility-benefits.cy.js

```diff
@@ -98,4 +114,74 @@ describe('Validate correct eligibility benefits display based on selected criter
         EN_DOLO_MOCK_DATA.data.benefits[22].benefit.eligibility[4].label
       )
   })
+
+  it('qa scenario 1 Covid EN - Verify correct benefit results for query values that includes covid in search parameter of URL', () => {
+    const benefitsCriteria = BENEFITS_ELIBILITY_DATA.scenario_1_covid.en.param
```

Comment:
Please remove this and follow the pattern in scenario3

https://github.com/GSA/px-benefit-finder/pull/902/files#diff-b58f7ff9ab058d9ab02fc76409896fe396763c9930bf33ddc0e2040ad63bdcc3R174-R178